src/views/Contributors.vue

```diff
@@ -1,12 +1,117 @@
 <template>
-  <div class="page">
-    Contributors page
+  <div class="view-container">
+    <div class="view-header">
+      <h1>Contributors</h1>
+    </div>
+    <div class="data-list contributors-data-list">
+      <div v-for="(item, index) in dataItems" :key="index" class="data-item contributors-data-item">
+        <div class="contributors-background"></div>
+        <div class="contributors-username">{{ item.username }}</div>
+        <div class="contributors-name">{{ item.name }}</div>
+        <div class="contributors-image"><img :src="item.image"/></div>
+        <div class="contributors-teamIds">
+          <Leaves :options="item.teamIds"/>
```

Comment:
@rabbitwerks how can we appropriately pass this `item.teamIds` to the Leaves component?

Reply:
you can change the name from options to just :teamIds="item.teamIds", item should probably be changed out to use contributor, its longer but more explicit in what it actually is, especially when the app will end up having a lot of "item" items lol